### PR TITLE
[LM-12753] Peer Testing Bugs

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
## Description
Fixing all peer testing bugs, and previously found bugs

**Features**

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [ ] Assignee added
- [ ] Reviewer added
- [ ] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lib/packages/core/lib/src/views/media/media_forwarding.dart [hunk 1]
```diff
@@ -76,6 +91,59 @@ class _LMChatMediaForwardingScreenState
...
+      children: [
+        Expanded(
+          child: _buildMainPreview(),
+        ),
+        if (mediaList.length > 1) _buildPreviewBar(),
+      ],
+    );
+  }
+
+  Widget _buildMainPreview() {
```
Comment (AnujLM) [inline lib/packages/core/lib/src/views/media/media_forwarding.dart:105]: rename _buildMainPreview() to _buildMediaPreview, and _buildMediaPreview() to _buildPreview()

File: lib/packages/core/lib/src/widgets/chatroom/chatroom_bar.dart [hunk 2]
```diff
@@ -738,7 +742,7 @@ class _LMChatroomBarState extends State<LMChatroomBar> {
       userText = 'You';
     }
     return LMChatBarHeader(
-      style: LMChatBarHeaderStyle.basic(),
+      style: LMChatBarHeaderStyle.basic().copyWith(height: 8.h),
```
Comment (AnujLM) [inline lib/packages/core/lib/src/widgets/chatroom/chatroom_bar.dart:745]: if we need 8.h height in our basic design then make it default in basic() method itself.

**Review by AnujLM — CHANGES_REQUESTED** [review]

File: lib/packages/core/lib/src/views/media/media_forwarding.dart (hunk 1 above)
Comment (divyanshgandhilm) [reply lib/packages/core/lib/src/views/media/media_forwarding.dart:105]: Kyon bhai? Ainvayin? Mauj mein?

Comment (AnujLM) [reply lib/packages/core/lib/src/views/media/media_forwarding.dart:105]: if you think _buildMediaPreview() and _buildMainPreview() is better then keep it.

File: lib/packages/core/lib/src/widgets/chatroom/chatroom_bar.dart (hunk 2 above)
Comment (AnujLM) [reply lib/packages/core/lib/src/widgets/chatroom/chatroom_bar.dart:747]: then there is no need of using copyWith() here, right?

**Review by AnujLM — APPROVED** [review]

**Pull request merged**